controllers/templatesync/template_sync.go

```diff
@@ -1026,7 +1026,7 @@ func (r *PolicyReconciler) cleanUpExcessTemplates(
 					})
 				}
 				// Log and ignore other errors to allow cleanup to continue since Gatekeeper may not be installed
-			} else if meta.IsNoMatchError(err) {
+			} else if discovery.IsGroupDiscoveryFailedError(errors.Unwrap(err)) {
```

Comment:
Should this be updated below for `v1` also?

Reply:
It should be there. Did GitHub now show the diff correctly for you?

Reply:
I don't see these lines changed in the PR:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/03bd852e4b4203506256317b4b9879fdbda5a105/controllers/templatesync/template_sync.go#L1062
https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/03bd852e4b4203506256317b4b9879fdbda5a105/controllers/templatesync/template_sync.go#L1430

Reply:
I see what you mean now. Fixed!